### ISSUE TITLE
Support for RTX 4060 ti 

### ISSUE THREAD

**Issue by Kaivalya192** [graspnet/anygrasp_sdk#67]
Nvidia GeForce RTX 4060 Ti has compute capability of 8.9 (Ada Lovelace architecture) and hence supports CUDA versions 11.8 to 12.2. (No support of 11.0 to 11.7.1).

how can i implement?

i have ubuntu 22.04..

**Comment by codepk37** [comment]
This env works on rtx 4060 and a4000 
conda list output is :

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
addict                    2.4.0                    pypi_0    pypi
anyio                     4.5.0              pyhd8ed1ab_0    conda-forge
arrow                     1.3.0              pyhd8ed1ab_0    conda-forge
asttokens                 2.4.1                    pypi_0    pypi
attrs                     24.2.0             pyh71513ae_0    conda-forge
autolab-core              1.1.1                    pypi_0    pypi
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
beautifulsoup4            4.12.3                   pypi_0    pypi
binaryornot               0.4.4                      py_1    conda-forge
blas                      1.0                         mkl  
blinker                   1.8.2                    pypi_0    pypi
bottleneck                1.3.7            py38ha9d4c09_0  
brotli                    1.1.0                hb9d3cd8_2    conda-forge
brotli-bin                1.1.0                hb9d3cd8_2    conda-forge
brotli-python             1.0.9            py38h6a678d5_8  
bzip2                     1.0.8                h5eee18b_6  
ca-certificates           2025.2.25            h06a4308_0  
cachetools                5.5.0              pyhd8ed1ab_0    conda-forge
certifi                   2024.8.30        py38h06a4308_0  
chardet                   5.2.0            py38h578d9bd_1    conda-forge
charset-normalizer        3.3.2              pyhd3eb1b0_0  
click                     8.1.7           unix_pyh707e725_0    conda-forge
colorama                  0.4.6              pyhd8ed1ab_0    conda-forge
colorlog                  6.9.0                    pypi_0    pypi
comm                      0.2.2              pyhd8ed1ab_0    conda-forge
configargparse            1.7                      pypi_0    pypi
contourpy                 1.1.1                    pypi_0    pypi
cookiecutter              2.6.0              pyhca7485f_0    conda-forge
cuda-cudart               11.7.99                       0    nvidia
cuda-cupti                11.7.101                      0    nvidia
cuda-libraries            11.7.1                        0    nvidia
cuda-nvrtc                11.7.99                       0    nvidia
cuda-nvtx                 11.7.91                       0    nvidia
cuda-runtime              11.7.1                        0    nvidia
cudatoolkit               11.8.0               h6a678d5_0  
cvxopt                    1.3.2                    pypi_0    pypi
cycler                    0.12.1             pyhd8ed1ab_0    conda-forge
cython                    3.0.11                   pypi_0    pypi
dash                      2.18.2                   pypi_0    pypi
dash-core-components      2.0.0                    pypi_0    pypi
dash-html-components      2.0.0                    pypi_0    pypi
dash-table                5.0.0                    pypi_0    pypi
debugpy                   1.6.7            py38h6a678d5_0  
decorator                 5.1.1              pyhd8ed1ab_0    conda-forge
dill                      0.3.9                    pypi_0    pypi
exceptiongroup            1.2.2              pyhd8ed1ab_0    conda-forge
executing                 2.1.0              pyhd8ed1ab_0    conda-forge
ffmpeg                    4.3                  hf484d3e_0    pytorch
filelock                  3.16.1             pyhd8ed1ab_0    conda-forge
flask                     3.0.3                    pypi_0    pypi
fonttools                 4.54.1                   pypi_0    pypi
freetype                  2.12.1               h4a9f257_0  
freetype-py               2.5.1                    pypi_0    pypi
future                    1.0.0                    pypi_0    pypi
gdown                     5.2.0                    pypi_0    pypi
giflib                    5.2.2                h5eee18b_0  
gmp                       6.2.1                h295c915_3  
gnutls                    3.6.15               he1e5248_0  
grasp-nms                 1.0.2                    pypi_0    pypi
h11                       0.14.0             pyhd8ed1ab_0    conda-forge
h5py                      3.11.0                   pypi_0    pypi
humanize                  4.10.0             pyhd8ed1ab_0    conda-forge
idna                      3.7              py38h06a4308_0  
imageio                   2.33.1           py38h06a4308_0  
importlib-metadata        8.5.0              pyha770c72_0    conda-forge
importlib_resources       6.4.5              pyhd8ed1ab_0    conda-forge
intel-openmp              2023.1.0         hdb19cb5_46306  
ipykernel                 6.29.5             pyh3099207_0    conda-forge
ipython                   8.12.3                   pypi_0    pypi
ipyvue                    1.11.2             pyhd8ed1ab_0    conda-forge
ipyvuetify                1.10.0             pyhd8ed1ab_0    conda-forge
ipywidgets                8.1.5              pyhd8ed1ab_0    conda-forge
itsdangerous              2.2.0                    pypi_0    pypi
jedi                      0.19.1             pyhd8ed1ab_0    conda-forge
jinja2                    3.1.4              pyhd8ed1ab_0    conda-forge
joblib                    1.4.2                    pypi_0    pypi
jpeg                      9b                   h024ee3a_2  
jsonschema                4.23.0             pyhd8ed1ab_0    conda-forge
jsonschema-specifications 2023.12.1                pypi_0    pypi
jupyter_client            8.6.3              pyhd8ed1ab_0    conda-forge
jupyter_core              5.7.2              pyh31011fe_1    conda-forge
jupyterlab_widgets        3.0.13             pyhd8ed1ab_0    conda-forge
kiwisolver                1.4.7                    pypi_0    pypi
krb5                      1.20.1               h568e23c_1  
lame                      3.100                h7b6447c_0  
lcms2                     2.12                 h3be6417_0  
ld_impl_linux-64          2.40                 h12ee557_0  
libbrotlicommon           1.1.0                hb9d3cd8_2    conda-forge
libbrotlidec              1.1.0                hb9d3cd8_2    conda-forge
libbrotlienc              1.1.0                hb9d3cd8_2    conda-forge
libcublas                 11.10.3.66                    0    nvidia
libcufft                  10.7.2.124           h4fbf590_0    nvidia
libcufile                 1.9.1.3                       0    nvidia
libcurand                 10.3.5.147                    0    nvidia
libcusolver               11.4.0.1                      0    nvidia
libcusparse               11.7.4.91                     0    nvidia
libedit                   3.1.20230828         h5eee18b_0  
libffi                    3.4.4                h6a678d5_1  
libgcc                    14.2.0               h77fa898_1    conda-forge
libgcc-ng                 14.2.0               h69a702a_1    conda-forge
libgfortran-ng            8.4.0               he6ba991_17  
libgfortran5              8.4.0               he6ba991_17  
libglvnd                  1.7.0                ha4b6fd6_2    conda-forge
libgomp                   14.2.0               h77fa898_1    conda-forge
libiconv                  1.16                 h5eee18b_3  
libidn2                   2.3.4                h5eee18b_0  
libnpp                    11.7.4.75                     0    nvidia
libnvjpeg                 11.8.0.2                      0    nvidia
libopenblas               0.3.2                h9ac9557_1  
libpng                    1.6.39               h5eee18b_0  
libsodium                 1.0.18               h7b6447c_0  
libstdcxx                 14.2.0               hc0a3c3a_1    conda-forge
libstdcxx-ng              11.2.0               h1234567_1  
libtasn1                  4.19.0               h5eee18b_0  
libtiff                   4.2.0                h85742a9_0  
libunistring              0.9.10               h27cfd23_0  
libuv                     1.48.0               h5eee18b_0  
libwebp                   1.2.0                h89dd481_0  
libwebp-base              1.2.0                h27cfd23_0  
lz4-c                     1.9.4                h6a678d5_1  
markdown                  3.6                pyhd8ed1ab_0    conda-forge
markdown-it-py            3.0.0              pyhd8ed1ab_0    conda-forge
markupsafe                2.1.5            py38h01eb140_0    conda-forge
matplotlib                3.7.5                    pypi_0    pypi
matplotlib-base           3.5.2            py38h826bfd8_0    conda-forge
matplotlib-inline         0.1.7              pyhd8ed1ab_0    conda-forge
mdurl                     0.1.2              pyhd8ed1ab_0    conda-forge
mesa                      2.4.0              pyhd8ed1ab_0    conda-forge
mesa-viz-tornado          0.1.3              pyhd8ed1ab_0    conda-forge
mesh-to-sdf               0.0.15                   pypi_0    pypi
minkowskiengine           0.5.4                    pypi_0    pypi
mkl                       2023.1.0         h213fc3f_46344  
mkl-service               2.4.0            py38h5eee18b_1  
mkl_fft                   1.3.8            py38h5eee18b_0  
mkl_random                1.2.4            py38hdb19cb5_0  
multiprocess              0.70.17                  pypi_0    pypi
munkres                   1.1.4              pyh9f0ad1d_0    conda-forge
nbformat                  5.10.4             pyhd8ed1ab_0    conda-forge
ncurses                   6.4                  h6a678d5_0  
nest-asyncio              1.6.0              pyhd8ed1ab_0    conda-forge
nettle                    3.7.3                hbbd107a_1  
networkx                  3.1                pyhd8ed1ab_0    conda-forge
ninja                     1.12.1               h06a4308_0  
ninja-base                1.12.1               hdb19cb5_0  
nomkl                     2.0                           0    anaconda
numexpr                   2.8.4            py38hc78ab66_1  
numpy                     1.23.4                   pypi_0    pypi
open3d                    0.18.0                   pypi_0    pypi
openblas-devel            0.3.2                         0    anaconda
opencv-python             4.10.0.84                pypi_0    pypi
openh264                  2.1.1                h4ff587b_0  
openssl                   1.1.1w               h7f8727e_0  
packaging                 24.2               pyhd8ed1ab_2    conda-forge
pandas                    1.5.3                    pypi_0    pypi
parso                     0.8.4              pyhd8ed1ab_0    conda-forge
pexpect                   4.9.0              pyhd8ed1ab_0    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pillow                    9.3.0            py38hace64e9_1  
pip                       24.2             py38h06a4308_0  
pkgutil-resolve-name      1.3.10             pyhd8ed1ab_1    conda-forge
platformdirs              4.3.6              pyhd8ed1ab_0    conda-forge
plotly                    5.24.1                   pypi_0    pypi
pointnet2                 0.0.0                    pypi_0    pypi
prompt-toolkit            3.0.48             pyha770c72_0    conda-forge
prompt_toolkit            3.0.48               hd8ed1ab_0    conda-forge
psutil                    6.0.0            py38hfb59056_0    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pure_eval                 0.2.3              pyhd8ed1ab_0    conda-forge
pyglet                    2.0.20                   pypi_0    pypi
pygments                  2.18.0             pyhd8ed1ab_0    conda-forge
pymdown-extensions        10.12              pyhd8ed1ab_0    conda-forge
pyopengl                  3.1.0                    pypi_0    pypi
pyparsing                 3.1.4              pyhd8ed1ab_0    conda-forge
pyquaternion              0.9.9                    pypi_0    pypi
pyrender                  0.1.45                   pypi_0    pypi
pyserial                  3.5                      pypi_0    pypi
pysocks                   1.7.1            py38h06a4308_0  
python                    3.8.18               h7a1cb2a_0  
python-dateutil           2.9.0.post0              pypi_0    pypi
python-fastjsonschema     2.20.0             pyhd8ed1ab_0    conda-forge
python-slugify            8.0.4              pyhd8ed1ab_0    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytorch                   1.13.0          py3.8_cuda11.7_cudnn8.5.0_0    pytorch
pytorch-cuda              11.7                 h778d358_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
pytz                      2024.2             pyhd8ed1ab_0    conda-forge
pywavefront               1.3.3                    pypi_0    pypi
pywavelets                1.4.1                    pypi_0    pypi
pyyaml                    6.0.2            py38h2019614_0    conda-forge
pyzmq                     25.1.2           py38h6a678d5_0  
reacton                   1.8.3              pyhd8ed1ab_0    conda-forge
readline                  8.2                  h5eee18b_0  
referencing               0.35.1             pyhd8ed1ab_0    conda-forge
requests                  2.32.3           py38h06a4308_0  
retrying                  1.3.4                    pypi_0    pypi
rich                      13.9.4             pyhd8ed1ab_0    conda-forge
rich-click                1.8.3              pyhd8ed1ab_0    conda-forge
rpds-py                   0.20.1                   pypi_0    pypi
ruamel-yaml               0.18.6                   pypi_0    pypi
ruamel-yaml-clib          0.2.8                    pypi_0    pypi
scikit-image              0.19.3                   pypi_0    pypi
scikit-learn              1.3.2                    pypi_0    pypi
scipy                     1.10.1                   pypi_0    pypi
setproctitle              1.3.3                    pypi_0    pypi
setuptools                75.1.0           py38h06a4308_0  
six                       1.16.0             pyh6c4a22f_0    conda-forge
sniffio                   1.3.1              pyhd8ed1ab_0    conda-forge
solara                    1.40.0             pyhd8ed1ab_0    conda-forge
solara-server             1.40.0             pyhff2d567_0    conda-forge
solara-ui                 1.40.0             pyhd8ed1ab_0    conda-forge
soupsieve                 2.6                      pypi_0    pypi
sqlite                    3.45.3               h5eee18b_0  
stack-data                0.6.3                    pypi_0    pypi
stack_data                0.6.2              pyhd8ed1ab_0    conda-forge
starlette                 0.41.2             pyha770c72_0    conda-forge
tbb                       2021.8.0             hdb19cb5_0  
tenacity                  9.0.0                    pypi_0    pypi
text-unidecode            1.3                pyhd8ed1ab_1    conda-forge
threadpoolctl             3.5.0                    pypi_0    pypi
tifffile                  2023.7.10                pypi_0    pypi
tk                        8.6.14               h39e8969_0  
torchaudio                0.13.0               py38_cu117    pytorch
torchvision               0.14.0               py38_cu117    pytorch
tornado                   6.4.1            py38hfb59056_0    conda-forge
tqdm                      4.67.0                   pypi_0    pypi
traitlets                 5.14.3             pyhd8ed1ab_0    conda-forge
transforms3d              0.3.1                    pypi_0    pypi
trimesh                   4.5.2                    pypi_0    pypi
types-python-dateutil     2.9.0.20241003     pyhff2d567_0    conda-forge
typing_extensions         4.11.0           py38h06a4308_0  
tzdata                    2024.2                   pypi_0    pypi
unicodedata2              15.1.0           py38h01eb140_0    conda-forge
urllib3                   2.2.3            py38h06a4308_0  
uvicorn                   0.32.1             pyh31011fe_0    conda-forge
watchdog                  4.0.1            py38h578d9bd_0    conda-forge
watchfiles                0.23.0           py38h4005ec7_0    conda-forge
wcwidth                   0.2.13             pyhd8ed1ab_0    conda-forge
websockets                13.0             py38h2019614_0    conda-forge
werkzeug                  3.0.6                    pypi_0    pypi
wheel                     0.44.0           py38h06a4308_0  
widgetsnbextension        4.0.13             pyhd8ed1ab_0    conda-forge
xz                        5.4.6                h5eee18b_1  
yaml                      0.2.5                h7f98852_2    conda-forge
zeromq                    4.3.5                h6a678d5_0  
zipp                      3.21.0             pyhd8ed1ab_0    conda-forge
zlib                      1.2.13               h5eee18b_1  
zstd                      1.4.9                haebb681_0  


hope it helps!
